docs/ui/components/CommandMenu/Items/CommandItemBase.tsx

```diff
@@ -0,0 +1,69 @@
+import { mergeClasses } from '@expo/styleguide';
+import { Command } from 'cmdk';
+import { useState } from 'react';
+
+import { openLink } from '../utils';
+
+import { Tag } from '~/ui/components/Tag';
+
+type Props = {
+  children: React.ReactNode;
+  url: string;
+  onSelect?: () => void;
+  isExternalLink?: boolean;
+  isNested?: boolean;
+  // Props forwarded to Command.Item
+  className?: string;
+  value?: string;
+};
+
+/**
+ * Wrapper for Command.Item that adds copy link on right/ middle click + visual copy indicator.
+ */
+export const CommandItemBase = ({
+  children,
+  url,
+  isExternalLink,
+  isNested,
+  onSelect,
+  className,
+  value,
+}: Props) => {
+  const [copyDone, setCopyDone] = useState(false);
+
+  const copyUrl = () => {
+    navigator.clipboard?.writeText(url);
+    setCopyDone(true);
+    setTimeout(() => setCopyDone(false), 1500);
+  };
+
+  return (
+    <Command.Item
+      className={mergeClasses(className, 'relative')}
+      value={value}
+      data-nested={isNested ? true : undefined}
+      onMouseUp={event => {
+        // middle click (typical *nix copy shortcut) and
+        // right click (works with Mac trackpads)
+        // NOTE: onAuxClick is not supported in Safari
```

Comment:
```suggestion
        // note(Keith): middle click (typical *nix copy shortcut) 
        // right click (works with Mac trackpads)
        // onAuxClick is not supported in Safari
```

Nittiest nit, but it's nice to have the comment attribution in code, since blame will not show the correct author of the comment after a while. 😉